Xamarin.Forms.Core/BindableObject.cs

```diff
@@ -605,7 +604,8 @@ enum BindableContextAttributes
 			IsBeingSet = 1 << 1,
 			IsDynamicResource = 1 << 2,
 			IsSetFromStyle = 1 << 3,
-			IsDefaultValue = 1 << 4
+			IsDefaultValue = 1 << 4,
+			IsInherited = 1 << 5,
```

Comment:
do we support Inherited BPs ? I don't see this flag used anywhere

Reply:
sorry thats leftover from me when I was doing something else with this